### ISSUE TITLE
Make it clear it's Kleros fork

### ISSUE THREAD

**Issue by marsrobertson** [kleros/uniswap-frontend#16]
About page should make it clear it's a Kleros fork, currently sending to uniswap.io

Update the footer and link to this repo, sending to GitHub of original Uniswap is MEH.

**Comment by clesaege** [comment]
This is a frontend to Uniswap. This is a fork of Uniswap.exchange but it is interacting with the Uniswap protocol.
It is linking to https://uniswap.io/ which is the Uniswap website (not that it is not a Uniswap.exchange website).

When we talked with Hayden he made really clear that Uniswap and uniswap.exchange are 2 different things.

**Comment by marsrobertson** [comment]
# TLDR my point

About page should tell this is a Kleros fork.

Maybe I don't understand something, let's ask at the source: https://twitter.com/marsxrobertson/status/1213990990832381952

---

![image](https://user-images.githubusercontent.com/44369284/71789032-f0c84e80-301f-11ea-8220-6c899a245781.png)

![image](https://user-images.githubusercontent.com/44369284/71789042-16555800-3020-11ea-8979-41baade7b80d.png)

![image](https://user-images.githubusercontent.com/44369284/71789063-384eda80-3020-11ea-9f0a-241d8b2f3710.png)

![image](https://user-images.githubusercontent.com/44369284/71789067-43a20600-3020-11ea-8956-abdde958fb12.png)

![image](https://user-images.githubusercontent.com/44369284/71789085-70561d80-3020-11ea-9215-fc38c5711c5b.png)

![image](https://user-images.githubusercontent.com/44369284/71789091-7815c200-3020-11ea-9b91-9814862a9188.png)

**Comment by marsrobertson** [comment]
### Asking at the source: https://twitter.com/marsxrobertson/status/1213990990832381952

https://twitter.com/marsxrobertson/status/1213990990832381952

![image](https://user-images.githubusercontent.com/44369284/71808651-458eb800-3066-11ea-9dcc-675976ea2fa9.png)
